experimental/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+__version__ = "0.1.4"
```

Comment:
is this needed?

Can you add some high level doc on:

1. what's this variant of pipeline
2. how to use it

Thanks for moving to the experimental dir. Really appreciate it. I think adding a bit doc and I think it is ready to go. @msbaines @blefaudeux, @anj-s what do you guys think?

Reply:
Yes, I will add a markdown file with results. 
We're still in the process of getting some more results. But let me add the current set.

Reply:
> is this needed?
I don't think so. Thanks for pointing it out.

Reply:
@min-xu-ai I added a docstring in the class AMPnetPipe. I also added instructions on how to run the benchmark script, experimental_ampnet.py